.github/workflows/integration-tests.yml

```
@@ -55,6 +55,11 @@ jobs:
           export RUNNER_OS=${{ runner.os }}
           QDRANT_VERSION='v1.7.4' ./tests/integration-tests.sh
         shell: bash
+      - name: Fastembed not installed tests
```

This phrasing was very confusing to me, maybe:
```suggestion
      - name: Run fastembed tests without fastembed
```